src/Compilers/CSharp/Test/Symbol/Symbols/Source/DelegateTests.cs

```diff
@@ -770,14 +770,14 @@ public static void Main()
             var model = compilation.GetSemanticModel(tree);
 
             ExpressionSyntax lambdaSyntax = tree.GetCompilationUnitRoot().DescendantNodes().OfType<ParenthesizedLambdaExpressionSyntax>().Single();
-            var lambda = (LambdaSymbol)model.GetSymbolInfo(lambdaSyntax).Symbol;
+            var lambda = (IMethodSymbol)model.GetSymbolInfo(lambdaSyntax).Symbol;
```

Comment:
Consider asserting the MethodKind, since we're no longer casting to LambdaSymbol.

Reply:
> Consider asserting the MethodKind, since we're no longer casting to LambdaSymbol.

I do not believe asserting this is the purpose of the test. I made similar changes to many tests.

---
In reply to: [339256077](https://github.com/dotnet/roslyn/pull/39498#discussion_r339256077) [](ancestors = 339256077)